.github/workflows/frontend-ci.yml

```diff
@@ -12,6 +12,9 @@ jobs:
   setup-job:
     runs-on: ubuntu-latest
     timeout-minutes: 3
+    permissions:
+      contents: read
+      pull-requests: read
```

Comment:
FYI:
The `dorny/paths-filter` action requires `pull-requests: read permission`.
https://github.com/dorny/paths-filter?tab=readme-ov-file#supported-workflows